shared/reducers/tracker2.tsx

```diff
@@ -109,14 +109,21 @@ export default Container.makeReducer<Actions, Types.State>(initialState, {
     })
   },
   [EngineGen.keybase1NotifyTrackingNotifyUserBlocked]: (draftState, action) => {
-    const {blocker, blocked} = action.payload.params.b
+    const {blocker, chatBlocked: _chatBlocked, followBlocked: _followBlocked} = action.payload.params.b
```

Comment:
I find this syntax pretty hard-to-read. How attached to it are you?

Reply:
not attached, removed